sunpy/spectra/tests/test_spectrogram.py

```diff
@@ -14,6 +14,16 @@
     Spectrogram, LinearTimeSpectrogram, _LinearView
 )
 
+from sunpy.spectra.spectrum import Spectrum
+
+def test_spectrum():
+    #The idea is to create two 1-D arrays, data and freq_axis,
+    #Pass them to the Spectrum object and check for equality of dimensions accordingly.
+    data = np.linspace(0, 100, 100)
+    freq = np.linspace(1, 100, 100)
+    spec = Spectrum(data,freq)
+    if not (np.shape(spec.data) == np.shape(spec.freq_axis)):
+        raise ValueError
```

Comment:
That is what I tried doing. When I do the equality check on the data and freq array, as shown in the example, everything works fine, their shapes are the same. But when I pass them to the spectrum object and do the equality check on spec.data and spec.freq_axis, their shapes come out different.